Mathlib/Algebra/Star/Basic.lean

```diff
@@ -299,18 +305,18 @@ def starRingEquiv [NonUnitalSemiring R] [StarRing R] : R ≃+* Rᵐᵒᵖ :=
 
 @[simp, norm_cast]
 theorem star_nat_cast [Semiring R] [StarRing R] (n : ℕ) : star (n : R) = n :=
-  (congr_arg unop (map_nat_cast (starRingEquiv : R ≃+* Rᵐᵒᵖ) n)).trans (unop_nat_cast _)
+  (congr_arg unop (map_natCast (starRingEquiv : R ≃+* Rᵐᵒᵖ) n)).trans (unop_natCast _)
 #align star_nat_cast star_nat_cast
 
-@[simp, norm_cast]
-theorem star_int_cast [Ring R] [StarRing R] (z : ℤ) : star (z : R) = z :=
-  (congr_arg unop <| map_int_cast (starRingEquiv : R ≃+* Rᵐᵒᵖ) z).trans (unop_int_cast _)
-#align star_int_cast star_int_cast
-
-@[simp, norm_cast]
-theorem star_rat_cast [DivisionRing R] [StarRing R] (r : ℚ) : star (r : R) = r :=
-  (congr_arg unop <| map_rat_cast (starRingEquiv : R ≃+* Rᵐᵒᵖ) r).trans (unop_rat_cast _)
-#align star_rat_cast star_rat_cast
+--@[simp, norm_cast]
+--theorem star_int_cast [Ring R] [StarRing R] (z : ℤ) : star (z : R) = z :=
+--  (congr_arg unop <| map_int_cast (starRingEquiv : R ≃+* Rᵐᵒᵖ) z).trans (unop_int_cast _)
+--#align star_int_cast star_int_cast
+--
+--@[simp, norm_cast]
+--theorem star_rat_cast [DivisionRing R] [StarRing R] (r : ℚ) : star (r : R) = r :=
+--  (congr_arg unop <| map_ratCast (starRingEquiv : R ≃+* Rᵐᵒᵖ) r).trans (unop_ratCast _)
+--#align star_rat_cast star_rat_cast
```

Comment:
Why is this commented out?

Reply:
@dupuisf May I ask you a reason for this?

Reply:
Uncommenting those lines gives a timeout. Still, I'll try to find a proof for those theorems, or ask for help if necessary.

Reply:
Yes, I temporarily commented them out while working on the file because they were extremely slow. These should definitely be revisited before merging.

Reply:
Giving unlimited time actually works, but I believe there is some alternative way to reduce time. Or is there?